extensions/provisioning/enable_auaditing_k3s.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+sudo mkdir -p -m o+w /var/lib/rancher/k3s/server/logs
+sudo install -m o+w /dev/null /var/lib/rancher/k3s/server/audit.yaml
+sudo chmod o+w /etc/systemd/system/k3s.service
+sed -i '$d' /etc/systemd/system/k3s.service
+sudo echo -e "       '--kube-apiserver-arg=audit-log-path=/var/lib/rancher/k3s/server/logs/audit.log'  \ \n       '--kube-apiserver-arg=audit-policy-file=/var/lib/rancher/k3s/server/audit.yaml'" >> /etc/systemd/system/k3s.service
```

Comment:
Just a clarifying question, what's the need for the spaces in the beginning of this command?

Reply:
That appears to be a copy paste error on my part.